### ISSUE TITLE
Component: Table - non-resizable columns are changing width when resizing other columns

### ISSUE THREAD

**Issue by victorsc** [primefaces/primeng#13765]
### Describe the bug

Given
```
<div class="card">
    <p-table [value]="products" stateStorage="local" stateKey="table" [resizableColumns]="true" columnResizeMode="expand" styleClass="p-datatable-gridlines" [tableStyle]="{'min-width': '50rem'}">
        <ng-template pTemplate="header">
            <tr>
                <th pResizableColumn [pResizableColumnDisabled]='true' [style.width]="'100px'">Code</th>
                <th pResizableColumn [pResizableColumnDisabled]='true' [style.width]="'100px'">Name</th>
                <th pResizableColumn>Category</th>
                <th pResizableColumn>Quantity</th>
            </tr>
        </ng-template>
        <ng-template pTemplate="body" let-product>
            <tr>
                <td [style.width]="'100px'">{{product.code}}</td>
                <td [style.width]="'100px'">{{product.name}}</td>
                <td>{{product.category}}</td>
                <td>{{product.quantity}}</td>
            </tr>
        </ng-template>
    </p-table>
</div>
```

when resizing `Category` or `Quantity` columns, `Code` and `Name` are getting wider.

The columns are not directly resizable and have a fixed width.

Please find below a reproduction repository for the issue.

May be similar with https://github.com/primefaces/primeng/issues/12344

Thanks you.

### Environment

    "primeng": "15.4.1",
    "@angular/core": "^15.2.8",

### Reproducer

https://github.com/victorsc/primeng-table-col-width

### Angular version

15.2.8

### PrimeNG version

15.4.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.17.1

### Browser(s)

Chrome 116

### Steps to reproduce the behavior

1. resize (expand) the Category or Quantity columns

Actual behaviour: Code and Name columns slightly change widths

### Expected behavior

Code and Name columns, which are not resisable, should not change width

**Comment by cetincakiroglu** [comment]
Could you please remove min-width and try again?

**Comment by victorsc** [comment]
hello @cetincakiroglu 
I believe you refer to the inline style `[style.width]="'100px'"`
Removing it doesn't help. The first 2 columns still get expended when I resize one of the last ones.

**Comment by cetincakiroglu** [comment]
Hi,

Your issue cannot be reproducible with the latest PrimeNG (16.4.1). See here: https://stackblitz.com/edit/s1j6jl?file=src%2Fapp%2Fdemo%2Ftable-column-resize-expand-mode-demo.html

The issue is probably fixed in a version within 15.4.1-16.4.1, we suggest you use the latest.

I am closing the issue since it does not exist with PrimeNG 16.4.1
